src/Controller/CaseController.php

```diff
@@ -38,10 +42,60 @@ public function summary(CaseEntity $case): Response
     /**
      * @Route("/{id}/information", name="case_information", methods={"GET"})
      */
-    public function information(CaseEntity $case): Response
+    public function information(CaseEntity $case, ComplaintCategoryRepository $categoryRepository): Response
     {
+        /** @var ComplaintCategory $complaintCategory */
+        $complaintCategory = $categoryRepository->findOneBy([
+            'board' => $case->getBoard(),
+            'municipality' => $case->getMunicipality(),
+            'name' => $case->getCaseType(),
+        ]);
```

Comment:
ComplaintCatetory seems to be case specific, so it should probably be a property on the CaseEntity class.

Reply:
Added complaintCategory property on the CaseEntity class.